features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/package.json

```diff
@@ -30,6 +30,7 @@
     "react-dom": "^15.5.4",
     "react-dropzone": "^3.13.3",
     "react-modal": "^2.3.2",
+    "react-notification-system": "^0.2.16",
```

Comment:
Adding `react-notification-system` as this is already used in  Ballerina [composer](https://github.com/ballerinalang/composer/blob/master/modules/web/package.json#L77)